charts/dgraph/templates/global-ingress.yaml

```diff
@@ -105,8 +105,7 @@ spec:
   {{- end }}
 {{- end }}
   rules:
-    - host: {{ .Values.global.ingress_grpc.alpha_grpc_hostname }}
-      http:
+    - http:
```

Comment:
so line 108 and `118-120` both had host?

Reply:
@nswamy Correct, host would appear twice.  Host should be omitted if not specified, then Kubernetes will insert `*` as the hostname.